internal/core/data/init.go

```diff
@@ -95,7 +94,11 @@ func BootstrapHandler(ctx context.Context, wg *sync.WaitGroup, startupTimer star
 		},
 		dataContainer.EventsChannelName: func(get di.Get) interface{} {
 			return chEvents
-		}})
+		},
+		dataContainer.HttpErrorHandlerName: func(get di.Get) interface{} {
+			return httpErrorHandler
```

Comment:
Why not eliminate the httpErrorHandler variable?

`return errorconcept.NewErrorHandler(loggingClient)`